release-notes/9.0/supported-os.md

```diff
@@ -0,0 +1,121 @@
+# .NET 9 - Supported OS versions
+
+[.NET 9](README.md) is a [Standard Term Support (STS)](../../release-policies.md) release and [is supported](../../support.md) on multiple operating systems per their lifecycle policy.
+
+For issues with .NET on operating systems not listed here, please open a GitHub issue in the appropriate .NET repository or contact the OS maintainer community. See [.NET Repos](../../Documentation/core-repos.md) for the repository list.
+
+## Windows
+
+OS                                    | Version                 | Architectures   | Lifecycle
+--------------------------------------|-------------------------|-----------------|----------
+[Windows 10 Client][Windows-client]   | Version 1607+           | x64, x86, Arm64 | [Windows][Windows-lifecycle]
+[Windows 11][Windows-client]          | Version 22000+          | x64, x86, Arm64 | [Windows][Windows-lifecycle]
+[Windows Server][Windows-Server]      | 2012+                   | x64, x86        | [Windows Server][Windows-Server-lifecycle]
+[Windows Server Core][Windows-Server] | 2012+                   | x64, x86        | [Windows Server][Windows-Server-lifecycle]
+[Nano Server][Nano-Server]            | Version 1809+           | x64             | [Windows Server][Windows-Server-lifecycle]
+
+[Windows-client]: https://www.microsoft.com/windows/
```

Comment:
Is there a better link than this? It is very salesy.

Reply:
there's an MS Learn one similar to what we use for Windows Server which looks better to me and doesn't fail the markdownlint check: https://github.com/dotnet/core/pull/9145